src/Rules/NoDebugInNamespace.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types=1);
+
+namespace Hihaho\PhpstanRules\Rules;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+
+/**
+ * @implements \PHPStan\Rules\Rule<\PhpParser\Node\Expr\FuncCall>
+ */
+class NoDebugInNamespace implements Rule
```

Comment:
Just to verify, this rule does not cover calls such as `Http::dd()->get('http://example.com')` (https://laravel.com/docs/11.x/http-client#dumping-requests) and `collect(['a', 'b'])->dd()` (https://laravel.com/docs/11.x/collections#method-dd) right?

Reply:
🤔 correct, probably best to add those instances as well